packages/components/Menu/src/MenuItemRadio/MenuItemRadio.android.tsx

```diff
@@ -0,0 +1,71 @@
+/** @jsx withSlots */
+import React from 'react';
+import { Image, Pressable, View } from 'react-native';
+
+import type { Slots, UseSlots } from '@fluentui-react-native/framework';
+import { compose, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { IconV1 as Icon } from '@fluentui-react-native/icon';
+import { TextV1 as Text } from '@fluentui-react-native/text';
+
+import { stylingSettings } from './MenuItemRadio.styling.android';
+import { useMenuItemRadio } from './useMenuItemRadio';
+import { getAccessibilityLabel, getTooltip } from '../MenuItem/MenuItem';
+import type {
+  MenuItemRadioProps,
+  MenuItemRadioSlotProps,
+  MenuItemRadioInfo,
+  MenuItemRadioType,
+} from '../MenuItemRadio/MenuItemRadio.types';
+
+export const menuItemRadioName = 'MenuItemRadio';
+
+export const MenuItemRadio = compose<MenuItemRadioType>({
```

Comment:
I think you'll also have to rewrite MenuItemRadio.tsx to use the new types you have so that the types match across the platforms. Right now, MenuItemRadio uses MenuItemCheckboxProps and MenuItemCheckboxSlotProps while the android version uses MenuItemRadioProps and MenuItemRadioSlotProps. I think they need to align to use MenuItemRadio types so that people on Android don't get typing errors.

Reply:
I think having a common types would be good idea. 
I approached this the previous version of this PR but that time radio was not separated out so didn't looked good. 

Now separated out everything related to Radio into /MenuRadioItem